README.md

```diff
@@ -63,7 +63,7 @@ OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.222-b10, mixed mode)
 
 Start Splice Machine first, e.g. `./start-splice-cluster -p cdh6.3.0 -bl`.
 Remove `-bl` options unless you are starting the Splice Machine instance for the very first time.
-For Spark 2.4, use cdh6.3.0 as the environment.  For Spark 2.3, use hdp3.1.0 .
+For Spark 3.0, use dbaas3.0 as the environment.  For Spark 2.4, use cdh6.3.0.  For Spark 2.3, use hdp3.1.0 .
```

Comment:
A table with all these version would make comprehension easier, wouldn't it?

Reply:
Changed in commit https://github.com/splicemachine/splice-machine-spark-connector/pull/31/commits/978249160be30c456b08c84acb8e6161476b9410